src/util/backtrace.cc

```diff
@@ -22,7 +22,9 @@
 module;
 #endif
 
+#ifdef SEASTAR_HAVE_EXECINFO
 #include <execinfo.h>
+#endif
```

Comment:
i think we can drop this `#include`. as this source file does not reference the symbols exposed by `execinfo.h`

Reply:
dropped it